modules/monitoring-about-accessing-monitoring-web-service-apis.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * monitoring/accessing-third-party-monitoring-uis-and-apis.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-accessing-monitoring-web-service-apis_{context}"]
+= About accessing monitoring web service APIs
+
+You can directly access web service API endpoints from the command line for the following monitoring stack components: Prometheus, Alertmanager, Thanos Ruler, and Thanos Querier.
```

Comment:
I think an unnumbered bullet list can be used here.

Reply:
Agree. Changed.